src/inputs/labeled-inputs.scss

```diff
@@ -28,169 +45,165 @@
       cursor: not-allowed;
     }
   }
+  // #endregion
 
-  > .iui-label {
-    margin-bottom: $iui-component-offset;
-    font-weight: $iui-font-weight-semibold;
-
-    &.iui-required {
-      &::after {
-        content: ' *';
-        @include themed {
-          color: t(iui-color-foreground-negative);
-        }
-      }
-    }
+  // Appropriate spacing even if no text label is given
```

Comment:
When these rules are needed? I removed them and nothing changed, tests are also passing.